javascript/builtins/String.json

```diff
@@ -1794,7 +1794,7 @@
                 "version_added": "1.0"
               },
               "webview_android": {
-                "version_added": "1"
+                "version_added": "41"
```

Comment:
This modifies `.replace()` instead of `.repeat()`. Line 1745 should be fixed instead.

Reply:
@saschanaz Thanks! I reverted the previous change and added another commit to this branch with the line 1745 change.